integration_tests/suite/base/test_call_pickup_target_user.py

```diff
@@ -104,6 +104,34 @@ def test_associate_multi_tenant(main_call_pickup, sub_call_pickup, main_user, su
     response.assert_match(400, e.different_tenant())
 
 
+@fixtures.call_pickup()
+@fixtures.user()
+@fixtures.user()
+@fixtures.user()
+def test_interceptor_user_has_targets(call_pickup, user1, user2, user3):
```

Comment:
Use the same semantic as other tests
```suggestion
def test_get_interceptor_user_relation(call_pickup, user1, user2, user3):
```